recipe_scrapers/wikicookbook.py

```diff
@@ -38,3 +38,25 @@ def instructions(self):
         return "\n".join(
             [normalize_string(instruction.get_text()) for instruction in instructions]
         )
+
+    def description(self):
+        paragraphs = list()
+        for tag in self.soup.find(class_="mw-parser-output"):
```

Comment:
Did you intend to use `find_all()` here? I think `find()` will only get you at most one result.

Reply:
It may look weird, but it is actually the correct behaviour to account for differences between books as they do not always seem to follow the same template.

`find()` returns an object of type `bs4.element.Tag` which is iterable, and the `for` loops iterates over the tags in the object. It's functionally similar to this:

```python3
>>> import recipe_scrapers
>>> bread = recipe_scrapers.scrape_me('https://en.wikibooks.org/wiki/Cookbook:Afghan_Bread')
>>> main_content = bread.soup.find(class_="mw-parser-output")
>>> for tag in main_content: print(tag.name)
table
None
p
p
[...]
```

Really, what I am doing is a shortcut for `for tag in self.soup.find("mw-parser-output").content`. I can add the `.content` if you think it makes it more explicit.

https://github.com/wention/BeautifulSoup4/blob/master/bs4/element.py#L907

Reply:
Ah, understood. I think it's fine, but yes I would have been expecting `.contents` or written it a different way to filter the `p` tags and show that I expect multiple tags here. You're code looks sound and I appreciate the thorough explanation here.

Reply:
Yeah, I wasn't thrilled by writing a loop with a `break`, but there's sadly not much in terms of identifying features, so no way to do `find_all("p")` just for the description. Thanks for the merge :)